packages/ppx/test/bucklescript/src/selectors_test.re

```diff
@@ -19,27 +19,27 @@ let testData = [
     CssJs.style(.
       [|
         CssJs.selector(.
-          {js|&:nth-child(even)|js},
+          {js|& :nth-child(even)|js},
           [|CssJs.color(CssJs.red)|]
         )
       |]
     ),
   ),
   (
     "nth-child(3n+1)",
-    [%cx "& > div:nth-child(3n+1) { color: blue; }"],
```

Comment:
Does it mean that it breaks the parser  if spaces aren't provided?

Reply:
No, this was just me editing the wrong string